app/src/organisms/ODD/QuickTransferFlow/utils/generateQuickTransferArgs.ts

```diff
@@ -338,17 +341,28 @@ export function generateQuickTransferArgs(
   const dropTipTrashBinLocationEntity = Object.values(
     invariantContext.trashBinEntities
   ).find(
-    entity => entity.location === quickTransferState.dropTipLocation.cutoutId
+    entity =>
+      'cutoutId' in quickTransferState.dropTipLocation &&
+      entity.location === quickTransferState.dropTipLocation.cutoutId
   )
   const dropTipWasteChuteLocationEntity = Object.values(
     invariantContext.wasteChuteEntities
   ).find(
-    entity => entity.location === quickTransferState.dropTipLocation.cutoutId
+    entity =>
+      'cutoutId' in quickTransferState.dropTipLocation &&
+      entity.location === quickTransferState.dropTipLocation.cutoutId
   )
-  const dropTipLocation =
-    dropTipTrashBinLocationEntity?.id ??
-    dropTipWasteChuteLocationEntity?.id ??
-    ''
+  const isReturnTip = isTipRackDropLocation(quickTransferState.dropTipLocation)
```

Comment:
can't you tell if its a return tip location based off of if the dropTip location is in the labware entities? so like


```suggestion
  const isReturnTip = invariantContext.labwareEntities[quickTransferState.dropTipLocation] != null
```